### PR TITLE
Idex 3081 Manage the autocompletion popup with javadoc

### PR DESCRIPTION
**Pull request by vitaliy-guliy**

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/javadoc/HTMLPrinter.java [hunk 1]
```diff
@@ -164,8 +164,8 @@ public static void insertPageProlog(StringBuffer buffer, int position, Color fgR
 	private static void appendColors(StringBuffer pageProlog, Color fgRGB, Color bgRGB) {
 		pageProlog.append("<body text=\""); //$NON-NLS-1$
 		appendColor(pageProlog, fgRGB);
-		pageProlog.append("\" bgcolor=\""); //$NON-NLS-1$
-		appendColor(pageProlog, bgRGB);
+//		pageProlog.append("\" bgcolor=\""); //$NON-NLS-1$
+//		appendColor(pageProlog, bgRGB);
```
Comment (skabashnyuk) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/javadoc/HTMLPrinter.java]: If this code is unused. Please remove it

**Comment by skabashnyuk** [comment]
Please make sure images is ours.

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java [hunk 2]
```diff
@@ -137,11 +145,12 @@ public static String insertStyle(final JavaResources javaResources, final String
...
-    public static ImageResource getImage(final String image) {
-        if (image == null) {
-            return null;
+    public static Icon getIcon(final String image) {
+        if (svgs.containsKey(image)) {
+            return new Icon("", svgs.get(image));
         }
-        return images.get(image);
+
+        return new Icon("", images.get(image));
```
Comment (svor) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java]: I think we can simplify it
return svgs.containsKey(image) ? new Icon("", svgs.get(image)) : new Icon("", images.get(image));

Comment (vitaliy-guliy) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java]: Look at the whole method please

**Comment by vitaliy-guliy** [comment]
Stevan said all those icons are ours.

File: plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/javadoc/HTMLPrinter.java (hunk 1 above)
Comment (vitaliy-guliy) [inline plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/javadoc/HTMLPrinter.java]: Done

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java (hunk 2 above)
Comment (svor) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java]: Why we can't replace
public static Icon getIcon(final String image) {
        if (svgs.containsKey(image)) {
            return new Icon("", svgs.get(image));
        }

        return new Icon("", images.get(image));
 }

to 

public static Icon getIcon(final String image) {
       return svgs.containsKey(image) ? new Icon("", svgs.get(image)) : new Icon("", images.get(image));  
 }

Comment (vitaliy-guliy) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistProcessor.java]: Sure we can. Do you think your variant is more readable and understandable?

**Pull request merged**